test/utils.js

```diff
@@ -48,6 +48,15 @@ export function checkValue(tensor, expected, nulp = 0) {
   }
 }
 
+export function checkValueByATOL(tensor, expected, tolerance) {
```

Comment:
```suggestion
export function checkValueByATOL(actual, expected, tolerance) {
```
`    const actualValue = tensor.getValueByIndex(i);`

To be clearer, as `tensor` and `expected` are both *tensors*.